### PR TITLE
Write redshift, not redshift_true, to instance catalogs

### PR DESCRIPTION
**Pull request by JoanneBogart**

### CODE REVIEW

**Comment by JoanneBogart** [comment]
The only change was to comment out the line overwriting 'redshift' with 'redshift_true'

**Comment by danielsf** [comment]
...ugh.... sorry about that

File: python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py [hunk 1]
```diff
@@ -493,7 +493,8 @@ def _transform_catalog(self, gc):
         """
         self._transform_object_coords(gc)
 
-        gc.add_quantity_modifier('redshift', gc.get_quantity_modifier('redshift_true'), overwrite=True)
+        #gc.add_quantity_modifier('redshift', gc.get_quantity_modifier('redshift_true'), overwrite=True)
```
Comment (jchiang87) [inline python/desc/sims/GCRCatSimInterface/DatabaseEmulator.py]: We should just delete this line instead of commenting it out.

**Review by jchiang87 — CHANGES_REQUESTED** [review]
Aside from my comment, this looks good to me.  The noMW flux calculations using the instance catalogs made with this change show that the resulting fluxes match to high precision the corresponding truth catalog values.

**Pull request merged**